scripts/src/machine.lua

```diff
@@ -4881,3 +4881,16 @@ if (MACHINES["NS32082"]~=null) then
 		MAME_DIR .. "src/devices/machine/ns32082.h",
 	}
 end
+
+---------------------------------------------------
+--
+--@src/devices/machine/tsconfdma.h,MACHINES["TSCONF"] = true
+---------------------------------------------------
+if (MACHINES["TSCONF"]~=null) then
+	files {
+		MAME_DIR .. "src/devices/machine/tsconfdma.cpp",
+		MAME_DIR .. "src/devices/machine/tsconfdma.h",
+		MAME_DIR .. "src/devices/machine/pckeybrd.cpp",
+		MAME_DIR .. "src/devices/machine/pckeybrd.h",
```

Comment:
You shouldn’t be putting these here – follow-in dependencies are identified correctly if you’re including the right headers.